### PR TITLE
Finish integration of Ansible installer

### PR DESCRIPTION
**Pull request by samirjorina**

### CODE REVIEW

File: crowbar_framework/app/views/barclamp/monasca/_edit_attributes.html.haml [hunk 1]
```diff
@@ -21,19 +21,3 @@
       = integer_field %w(metric_agent max_measurement_buffer_size)
       = integer_field %w(metric_agent backlog_send_rate)
       = integer_field %w(metric_agent amplifier)
-
-    %fieldset
```
Comment (kornicameister) [inline crowbar_framework/app/views/barclamp/monasca/_edit_attributes.html.haml:23]: I am not enforcing that, but I think I'd keep passwords here.

Comment (samirjorina) [reply crowbar_framework/app/views/barclamp/monasca/_edit_attributes.html.haml:23]: No idea what would be better...

**Review by kornicameister — COMMENTED** [review]
Generally, I managed to lunch installer, but there are some issues coming from ansible itself.
Let's try to resolve them first prior to merging that PR

File: chef/cookbooks/monasca/recipes/master.rb [hunk 2]
```diff
@@ -99,10 +73,48 @@
...
+  zookeeper_hosts: cmm_net_ip,
+  kafka_hosts: "#{cmm_net_ip}:9092",
+  mariadb_bind_address: cmm_net_ip,
+  database_host: cmm_net_ip,
+  monasca_api_url: "http://#{pub_net_ip}:8070/v2.0",
+  monasca_log_api_url: "http://#{pub_net_ip}:5607/v2.0",
+  memcached_nodes: "#{cmm_net_ip}:11211",
+  influxdb_url: "http://#{cmm_net_ip}:8086",
+  elasticsearch_nodes: "[#{cmm_net_ip}]",
+  elasticsearch_hosts: cmm_net_ip
```
Comment (kornicameister) [inline chef/cookbooks/monasca/recipes/master.rb:111]: I see no information about keystone_host ?

File: chef/cookbooks/monasca/recipes/master.rb [hunk 3]
```diff
@@ -99,10 +73,48 @@
...
+}.to_json
+
 execute "run ansible" do
-  command "rm -f /opt/fujitsu/monasca-installer/.installed"\
-          "&& ansible-playbook -i cmm-hosts monasca.yml"\
-          "&& touch /opt/fujitsu/monasca-installer/.installed"
-  cwd "/opt/fujitsu/monasca-installer"
+  command "rm -f /opt/monasca-installer/.installed"\
+          "&& ansible-playbook -i cmm-hosts -e '#{ansible_vars}' monasca.yml"\
```
Comment (kornicameister) [inline chef/cookbooks/monasca/recipes/master.rb:116]: Should cmm-hosts be left in installer directory after install ?
I wanted to debug kibana installation, but couldn't locate that file.

**Review by kornicameister — CHANGES_REQUESTED** [review]

**Pull request merged**